src/clarity-angular/datagrid/_datagrid.clarity.scss

```diff
@@ -39,6 +41,11 @@
         min-height: 0;
         // IE10 (maybe later versions too) needs this
         overflow: hidden;
+
+        &.has-filter {
+            // space for (filter padding + 2 * 24 (close button + input field) + margin top + Header height + 1px for vertical rhythm)
+            min-height: $clr-datagrid-filter-padding * 2 + $clr_baselineRem_2 + $clr-datagrid-filter-top-margin + $clr_baselineRem_1_5 + 1;
```

Comment:
This only works for our string filters. It doesn't work for any kind of custom filter, see our colorpicker filter for instance.